src/components/DateRangePicker.js

```diff
@@ -307,256 +414,359 @@ class DateRangePicker extends React.Component {
+                                focusedDay={
+                                  this.state.focusedDay ||
+                                  this.state.currentDate
+                                }
+                                getDateRangePosition={
+                                  this._getDateRangePosition
+                                }
+                                handleDateHover={this._handleDateHover}
+                                handleDateSelect={date => {
+                                  this._handleDateSelect(date);
+                                  this.setState({ showCalendar: false });
+                                }}
+                                handleKeyDown={this._handleDayKeyDown}
+                                isInActiveRange={this._isInActiveRange}
+                                minimumDate={this.props.minimumDate}
+                                selectedEndDate={this.state.selectedEndDate}
+                                selectedStartDate={this.state.selectedStartDate}
+                                styles={styles}
+                              />
                             </div>
-                          );
-                        })}
+                          </div>
+                        )}
                     </div>
-                    <MonthTable
-                      activeSelectDate={this.state.activeSelectDate}
-                      currentDate={this.state.currentDate}
-                      focusedDay={this.state.focusedDay || this.state.currentDate}
-                      getDateRangePosition={this._getDateRangePosition}
-                      handleDateHover={this._handleDateHover}
-                      handleDateSelect={this._handleDateSelect}
-                      handleKeyDown={this._handleDayKeyDown}
-                      isInActiveRange={this._isInActiveRange}
-                      minimumDate={this.props.minimumDate}
-                      selectedEndDate={this.props.selectedEndDate}
-                      selectedStartDate={this.props.selectedStartDate}
-                      styles={styles}
-                    />
-                    {!isLargeOrMediumWindowSize && (
-                      <div style={styles.applyButton}>
+
+                    {!this.state.showCalendar ? (
+                      <div style={styles.bottomPane}>
+                        <Button
+                          onClick={() => {
+                            this.setState({
+                              focusedDay: this.props.selectedStartDate,
+                              selectedStartDate: this.props.selectedStartDate,
+                              selectedEndDate: this.props.selectedEndDate
+                            });
+                            this._handleScrimClick();
+                          }}
+                          type='secondary'
+                        >
+                          Cancel
```

Comment:
I can already tell that this button is not descriptive enough for accessibility.  We should probably add an aria label with a better description than just `Cancel`